client/checkout/index.js

```diff
@@ -393,24 +396,24 @@ jQuery( function ( $ ) {
 	 * Displays the authentication modal to the user if needed.
 	 */
 	function maybeShowAuthenticationModal() {
-		var partials = window.location.hash.match(
+		const partials = window.location.hash.match(
 			/^#wcpay-confirm-pi:(.+):(.+):(.+)$/
 		);
 
 		if ( ! partials ) {
 			return;
 		}
 
-		var orderPayIndex = document.location.href.indexOf( 'order-pay' );
-		var isOrderPage = orderPayIndex > -1;
+		const orderPayIndex = document.location.href.indexOf( 'order-pay' );
```

Comment:
I know this change is not part of this PR and that modern browsers map `document.location` to `window.location`, but I think it would be nice to use `window.location` here to conform with how we use it in other places. What do you think?

Reply:
Absolutely, I hadn't even noticed it. I checked and there are no other usages of `document.location`. Fixed in https://github.com/Automattic/woocommerce-payments/pull/902/commits/5be2767f7e3fbb657839564e212b0e8bb69b2cb5